modules/addons/src/gltf/gltf-environment.js

```diff
@@ -1,11 +1,13 @@
 import GL from '@luma.gl/constants';
-import {Texture2D, TextureCube, loadImage} from '@luma.gl/core';
+import {Texture2D, TextureCube} from '@luma.gl/core';
+import {loadImage} from '@loaders.gl/images';
```

Comment:
Agree we should shift to loaders.gl, but FWIW we still have `loadImage` in luma.gl: https://github.com/uber/luma.gl/search?q=loadImage&unscoped_q=loadImage

Reply:
The luma version is browser-only. That's why I had to switch.

As an aside, I think this is a really nice selling point for loaders: that you can write unit tests for browser code and just not worry about it. Also thinking if we end up presenting our testing utils, loaders.gl could be part of a full WebGL testing solution.